workers/unity/Packages/com.improbable.gdk.core/Worker/SerializedMessagesToSend.cs

```diff
@@ -37,8 +37,11 @@ public class SerializedMessagesToSend
         private readonly List<IComponentSerializer> componentSerializers = new List<IComponentSerializer>();
         private readonly List<ICommandSerializer> commandSerializers = new List<ICommandSerializer>();
 
+        private readonly UpdateParameters updateParams = new UpdateParameters();
+
         public SerializedMessagesToSend()
         {
+            updateParams.Loopback = ComponentUpdateLoopback.None;
```

Comment:
Can this not be set in the object's constructor?

Reply:
It doesn't have a non default constructor.

Reply:
If its a struct you can just use named initialisation.

Reply:
It's a class. It has to be set after it's definition.